python/runtime/cudaq/algorithms/py_sample.cpp

```diff
@@ -23,6 +23,13 @@
 
 namespace cudaq {
 
+/// @brief For asynchronous execution, we need to construct OpaqueArguments
```

Comment:
nit: This documentation string isn't exactly "brief". heh

(I think the \brief tag is meant to be a very short line that fits in the summary table of the doxygen.

Reply:
I've updated these doxygen comments to separate the explanation text from the brief headline.